### ISSUE TITLE
Warning this request accesses system indices when reindexing

### ISSUE THREAD

**Issue by GMolini** [ankane/searchkick#1463]
Im getting the following warning when i do myModel.reindex
```
warning: 299 Elasticsearch-7.10.1-1c34507e66d7db1211f66f3513706fdf548736aa 
"this request accesses system indices: [.kibana_task_manager_1, .apm-custom-link, .apm-agent-configuration, .kibana_1], 
but in a future major version, direct access to system indices will be prevented by default"
```

It still works fine, just wanted to give a heads up.

**Comment by ankane** [comment]
Hey @GMolini, thanks for reporting. Fixed on master.

**Comment by mike1011** [comment]
Hi @ankane , Thanks for this awesome gem. I am facing this problem while using elasticsearch 7+.
As the latest version requires, I have setup the elastic user credentials and using it , since then I started facing this problem.

.Sharing more details below - 

```
##searchkick.rb
Searchkick.client_options = {
  url: "http://elastic:myelasticpassword@localhost:9200",
  retry_on_failure: true,
  transport_options:
    { request: { timeout: 250 } }
}

Searchkick.redis = ConnectionPool.new { Redis.new }

```
```
##Gemfile
gem "searchkick"

```
```
##Gemfile.lock
Using `searchkick (3.1.3)`
```

```
##rails console

warning: 299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default"
warning: 299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default"
Reindexing User...
warning: 299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default"
warning: 299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default"
Reindex complete
```

Let me know if you need some more details from me.Thanks.

**Comment by ankane** [comment]
Hey @mike1011, try updating Searchkick. The fix is in version 4.4.3+.
